playbooks/templates/.env.j2

```diff
@@ -176,19 +176,20 @@ SKYNET_ACCOUNTS_LOG_LEVEL={{ webportal_common_config.accounts_log_level }}
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
 # REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string | lower }}
+{% endif %}
 
-{% if webportal_common_config.accounts_limit_access | string | lower != 'false' %}
-# Account Test account for health checks.
+{% if webportal_common_config.accounts_limit_access | default('') | string | lower != 'false' %}
```

Comment:
nit: This `default` should be `authenticated` in order to be consistent with `playbooks/tasks/portal-configs-load.yml`. But the result is the same, therefore this is a `nit` and not ` req`.

Reply:
@ro-tex here we are not setting the value of `acconuts_limit_access`, we are checking it. That is why the default is blank, Because we don't need to set it here or in ansible.